docs/ref/contrib/gis/install/spatialite.txt

```diff
@@ -56,45 +56,18 @@ __ http://www.sqlite.org/download.html
 
 .. _spatialitebuild:
 
-SpatiaLite library (``libspatialite``) and tools (``spatialite``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+SpatiaLite library (``libspatialite``)
```

Comment:
Could you separate the changes in this file into a separate commit so it can be more easily backported?